src/nxt_fs.c

```diff
@@ -273,6 +273,32 @@ nxt_fs_mkdir_all(const u_char *dir, mode_t mode)
 }
 
 
+nxt_int_t
+nxt_fs_mkdir_dirname(const u_char *path, mode_t mode)
+{
+    char       *ptr;
+    char       *dir;
```

Comment:
You could just compress the declaration of ptr and dir into a single line.  That's common here.

Reply:
Ah, so it is...